application.properties/metasfresh/backend/metasfresh-dist/serverRoot/application.properties

```diff
@@ -9,7 +9,7 @@ server.port=8181
 # these properties are local to my dev environment
 #
 spring.rabbitmq.host=localhost
-spring.rabbitmq.port=30050
+spring.rabbitmq.port=5672
```

Comment:
I see that all of these `application.properties` are changed and can no longer be used with the current k8s stack. Do we intend to completely remove k8s? If not i think we should not touch the current (k8s) `applicaton.properties` files.

Reply:
If we want to keep both (why?) we should make 1 folder for each different stack.

Reply:
Lastly, what about non-docker users? ie ruxi/teo? afaik they have postgres and rabbitmq installed locally. for them it will be even trickyer to use elk, as they have nothing "standard" (ie like we, with k8s have)

Reply:
yes, good point.
i intend to switch us all to docker-compose, but ofc it's requires some talking, maybe a workshop and so on.
btw, why: because "standardisation of dev-environments". e.g. all devs shall work with the same postgres-version that we also have out there

(btw note that 5672 is the rabbitmq standard port, so teo ruxi etc should have it)

Reply:
PS:

> i intend to switch us all to docker-compose

because we don't really use any of the nice things that k8s brings to the table, but we pay for the complexity